### PR TITLE
WIP: Improve builder

### PR DESCRIPTION
**Pull request by LanderU**

### CODE REVIEW

**Comment by LanderU** [comment]
FYI @carlossvg

**Review by carlossvg — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -40,6 +40,15 @@ ARG UNAME_R
...
 ARG triple=aarch64-linux-gnu
 
+# setup environment
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV ARCH ${ARCH}
+ENV UNAME_R ${UNAME_R}
+ENV triple "aarch64-linux-gnu"
+ENV CROSS_COMPILE "${triple}-"
+ENV LANG "C fakeroot debian/rules printenv"
```
Comment (razr) [inline Dockerfile:50]: ```fakeroot debian/rules printenv``` was not export, it was just a command

**Review by razr — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (LanderU) [reply Dockerfile:50]: @razr, sorry for the silence! You're right I will fix it.

**Comment by razr** [comment]
@LanderU do we still need to work on this PR or we can close it?

**Comment by LanderU** [comment]
@razr, no, closing here.

**Pull request closed without merging**